### DISCUSSION TITLE
building with make run

### DISCUSSION THREAD

**Discussion by wasimahmad2022eurustechnologies** [goauthentik/authentik#5198] [Q&A]
i want to build the **/web** on my local machine on official docs it is written that we can build the app with **"make run"** but when i am running this there is no **"make run"** command in make file, can anyone  **help** about?

**link to docs.**
https://version-2022-12.goauthentik.io/developer-docs/setup/full-dev-environment

**Comment by BeryJu** [comment]
When referring to the versioned docs, make sure you have the correct branch checked out. For this case, check out the branch `version-2022.12`. Otherwise, you can check out the `main` branch, where the command is now `ak server` instead of `make run`

**Reply by wasimahmad2022eurustechnologies** [reply]
can you tell me how to install **ak** utility as by default it is not available?

**Reply by BeryJu** [reply]
you can use the `ak` tool when you have the python virtualenv enabled, otherwise you can run `./lifecycle/ak`

**Reply by wasimahmad2022eurustechnologies** [reply]
can you tell me where is the requirement-dev.txt file?

**Reply by BeryJu** [reply]
Authentik uses poetry to manage dependencies, see https://goauthentik.io/developer-docs/setup/full-dev-environment

**Reply by wasimahmad2022eurustechnologies** [reply]
thanks for your help, just runt the server as there are many commands, please tell me how to start both the front end and back end (local machine)?

also if you could tell about bundling the front end.
